docs/product-installation-guide/src/main/asciidoc/controller-jws-install-proc.adoc

```diff
@@ -0,0 +1,50 @@
+[id='controller-standalone-jws-install-proc']
+= Installing the Controller on Red Hat JBoss Web Server
+
+You can install the Controller and use the REST API or the {KIE_SERVER} Java Client API to interact with it.
```

Comment:
I think that this sentence is a bit confusing because in 6.x series there was a Java client for Decision Central, but it had different functionality.
I think there should be something like "Java client for Controller"